project/tests/test_mobile_example.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from common.facade import logger
+from project.api.endpoints.daily_weather_endpoint import DailyWeatherEndpointBuilder
+from project.mobile.steps.global_steps import add_location
+from project.mobile.steps.navigation_steps import navigate_to_main_page
+
+
+@pytest.mark.usefixtures("start_mobile_session")
+@pytest.mark.mabile_sample
```

Comment:
mobile_sample?

Reply:
Yeap, it's one more way to run tests using custom mark

Reply:
No, I meant typo) mabile_sample != mobile_sample =)